test/writeObfuscatedMockRac.ts

```diff
@@ -0,0 +1,35 @@
+import * as fs from 'fs';
+
+import { encodeBase64Hash, getMD5Hash } from '../src/obfuscation';
+
+import {
+  OBFUSCATED_MOCK_RAC_RESPONSE_FILE,
+  TEST_DATA_DIR,
+  readMockRacResponse,
+} from './testHelpers';
+
+export function generateObfuscatedMockRac() {
```

Comment:
This function gets called when you run `yarn obfuscated-mock-rac` to re-generate the obfuscated RAC file